docs/install/openshift.md

```diff
@@ -36,6 +36,45 @@ oc login -u kubeadmin
 By default, the operator is installed into the `argocd-operator-system` namespace. To modify this, update the
 value of the `namespace` specified in the `config/default/kustomization.yaml` file. 
 
+### Conversion Webhook Support
+
+ArgoCD `v1alpha1` CRD has been **deprecated** starting from **argocd-operator v0.8.0**. To facilitate automatic migration of existing `v1alpha1` ArgoCD CRs to `v1beta1`, conversion webhook support has been introduced.
+
+By default, the conversion webhook is disabled for the manual installation of the operator. Users can modify the configurations to enable conversion webhook support using the instructions provided below.
+
+!!! warning
+    Enabling the webhook is optional. However, without conversion webhook support, users are responsible for migrating any existing ArgoCD v1alpha1 CRs to v1beta1.
```

Comment:
maybe we should mention that it is enabled out of the box
https://github.com/argoproj-labs/argocd-operator/pull/964/files#diff-29597131adc5aaecd3350041d098624e877013a910db6312107dadfaa0a5690eR1750